cpp/src/parquet/thrift_internal.h

```diff
@@ -332,6 +408,35 @@ static inline format::SortingColumn ToThrift(SortingColumn sorting_column) {
   return thrift_sorting_column;
 }
 
+static inline format::GeospatialStatistics ToThrift(
+    const geospatial::EncodedGeoStatistics& encoded_geo_stats) {
+  format::GeospatialStatistics geospatial_statistics;
+
+  geospatial_statistics.__set_geospatial_types(encoded_geo_stats.geospatial_types);
+
+  format::BoundingBox bbox;
+  if (encoded_geo_stats.xy_bounds_present) {
```

Comment:
Can move the `bbox` instantiation inside the `if`.
```suggestion
  if (encoded_geo_stats.xy_bounds_present) {
    format::BoundingBox bbox;
```